packages/big-design/src/components/Table/HeaderCell/HeaderCell.tsx

```diff
@@ -0,0 +1,68 @@
+import { ArrowDownwardIcon, ArrowUpwardIcon } from '@bigcommerce/big-design-icons';
+import React, { memo } from 'react';
+
+import { Flex } from '../../Flex';
+
+import { StyledTableHeaderCell, StyledTableHeaderCheckbox } from './styled';
+
+export interface HeaderCellProps extends React.TableHTMLAttributes<HTMLTableCellElement> {
+  align?: 'left' | 'center' | 'right';
+  isCheckbox?: boolean;
+  isSortable?: boolean;
+  isSorted?: boolean;
+  sortDirection?: 'ASC' | 'DESC';
+  stickyHeader?: boolean;
+  width?: number | string;
+  onSortClick?(): void;
+}
+
+export const HeaderCell: React.FC<HeaderCellProps> = memo(
+  ({
+    align,
+    children,
+    isCheckbox,
+    isSortable,
+    isSorted,
+    onSortClick,
+    sortDirection,
+    stickyHeader,
+    width,
+  }: HeaderCellProps) => {
+    const renderSortIcon = () => {
+      if (!isSorted) {
+        return null;
+      }
+
+      return sortDirection === 'ASC' ? (
+        <ArrowUpwardIcon size="medium" data-testid="asc-icon" />
+      ) : (
+        <ArrowDownwardIcon size="medium" data-itestid="desc-icon" />
```

Comment:
You may want to add `title` props to these SVG's

Reply:
Good catch.